src/get-active-task-id-from-summary.js

```diff
@@ -0,0 +1,22 @@
+export const getActiveTaskIdFromSummary = taskSummaryResponse => {
```

Comment:
This was taken from https://github.com/dhis2/data-administration-app/blob/patch/2.36.3/src/pages/analytics/Analytics.js#L229-L253